### DISCUSSION TITLE
online class

### DISCUSSION THREAD

**Discussion by rootsmusic** [thonny/thonny#3022] [General]
Is Thonny being used by any online class, especially in English?

**Comment by aivarannamaa** [comment]
Programiz used to bring lots of traffic to Thonny's homepage. I haven't found another simple cookieless solution after old Google Analytics was turned off, so I don't know the current situation, but I believe there are more online courses using Thonny. Currently there are about 8000 downloads per day from GitHub.

**Comment by selmen2004** [comment]
In Tunisia , Thonny is the official programming editor for all secondary students since 2018 , about 300k -400k students each year

**Comment by Umardeshmukh** [comment]
I vsitied thony first time and I saw the heading that is in support of Ukrain ,that's greate guesture by developer, But It would have been so much better if this empathy was also evident for the innocent children of Palestine."

**Comment by JaysonA0522** [comment]
Hello @rootsmusic! Made an account to answer your question, and hopefully dive into Github more. 

I'm a transfer student at Arizona State University in the United States. Just this week I began a new course, dealing with programming logic. Our professor has advised all students to download & utilize Thonny exclusively for all semester assignments. It's listed under "required software" in our syllabus. 

I think it's great we're being told to use an open source program - it's fun to explore the code that runs the application! Hope this helps. 
<img width="803" alt="image" src="https://github.com/thonny/thonny/assets/59128261/d2e61160-4994-4c24-9a79-cbf38f36bca0">
